driver/razerkbd_driver.h

```diff
@@ -94,6 +94,7 @@
 #define USB_DEVICE_ID_RAZER_DEATHSTALKER_V2_PRO_WIRED 0x0292
 #define USB_DEVICE_ID_RAZER_DEATHSTALKER_V2 0x0295
 #define USB_DEVICE_ID_RAZER_BLACKWIDOW_V3_TK 0x0A24
+#define USB_DEVICE_ID_RAZER_BLACKWIDOW_V4_PRO 0x028D
```

Comment:
please keep this list sorted by USB PID, so put this just below USB_DEVICE_ID_RAZER_BLADE_14_2022

Reply:
Fixed in 1a9965bc346bb01841ffa1216f09b26d3ed00867